### ISSUE TITLE
CheckBox control does not support UIA accessibility

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#3227]
* .NET Core Version: 

.NET Core 5.0
Version: 5.0.100-alpha.1.20073.10
Commit: 29f4d693a9
Runtime Environment:
OS Name: Windows
OS Version: 10.0.18363
OS Platform: Windows
RID: win10-x64
Base Path: C:\Program Files\dotnet\sdk\5.0.100-alpha1-05536
Host (useful for support):
Version: 5.0.0-alpha.1.20072.3
Commit: c3dc1fdfdc

* Have you experienced this same bug with .NET Framework?: 

Yes

**Problem description:**

![image](https://user-images.githubusercontent.com/23213980/81215671-56364700-8fe2-11ea-9183-7088f188c9f1.png)

**Accessibility providers:**
[pid:15032,providerId:0x40CA8 Main:Nested [pid:12228,providerId:0x40CA8 Main(parent link):Microsoft: MSAA Proxy (unmanaged:UIAutomationCore.dll)];
Hwnd(parent link):Microsoft: HWND Proxy (unmanaged:uiautomationcore.dll)]"

**Expected behavior:**
There should be UIA accessibility provider from WinForms

**Minimal repro:**
Create WinForms app with CheckBox control and analyze it using Inspect.

**Related item:**
This work item is the sub of #2164

**Comment by Jessie-Zhang01** [comment]
Verified the issue with latest NET SDK 6.0.100-preview.3.21154.10, this issue is fixed. Please see following screenshot:
![image](https://user-images.githubusercontent.com/45864649/110078754-2eab3900-7dc3-11eb-8918-29485e5fea8f.png)
